### PR TITLE
Made edits to the scenario

### PR DESCRIPTION
**Pull request by namzo198**
Testing first pull and merge request

### CODE REVIEW

File: src/components/Scenario/index.js [hunk 1]
```diff
@@ -9,62 +9,27 @@ function Scenario() {
...
+      <p className="scenario-text">You are <strong><ProfileLink name="alex_doe"/> </strong> from Fresno. California and regularly use FriendBook ( a social media site) for professional and leisure activities. You are planning on applying for a job, go through the posts you have made in the recent past and see if you are okay with them.</p>
 
-      <p className="scenario-task">Given that you have an upcoming interview, go through your profile and see if you are okay with what&apos;s on it.</p>
+      <p className="scenario-task">For each of your posts, you could consider taking any of the following actions based on how you feel or think the post would reflect on you to a potential employer.</p>
```
Comment (henryksloan) [inline src/components/Scenario/index.js]: > you could consider

Is this wording too passive? Should we say "you *should* consider"?

File: src/components/Scenario/index.js [hunk 2]
```diff
@@ -9,62 +9,27 @@ function Scenario() {
...
-          <label>Write a post stating that you are looking for a job</label>
-          <span className="tooltiptext"> It’s not uncommon for people to find a job via their social network. Write a post that states your looking for a job to notify your friends about your search. To avoid awkward situations, though, it is probably best not to tell anyone at your current employer about it</span>
+          <label>Deleting the post</label>
+          <span className="tooltiptext"> It’s not uncommon for people to delete posts that they think would reflect poorly on them.</span>
         </li>
 
         <li className="scenario-list-item tooltip">
-          <label>Review your contact information</label>
-          <span className="tooltiptext">Make sure you have a professional email address listed (alexdoe@gmail.com) where recruiters can contact you</span>
+          <label>Change your post’s audeince </label>
```
Comment (henryksloan) [inline src/components/Scenario/index.js]: audeince -> audience

File: src/components/Scenario/index.js [hunk 3]
```diff
@@ -9,62 +9,27 @@ function Scenario() {
...
+          <span className="tooltiptext"> It’s not uncommon for people to change the audience of their post to control who can view/access their posts.</span>
```
Comment (henryksloan) [inline src/components/Scenario/index.js]: post -> posts

to match "It’s not uncommon for people to delete posts"

File: src/components/Scenario/index.js [hunk 4]
```diff
@@ -9,62 +9,27 @@ function Scenario() {
...
         </li>
 
         <li className="scenario-list-item tooltip">
-          <label>Review your profile page</label>
-          <span className="tooltiptext">Research shows that recruiters can be biased by things on candidates’ FriendBook profile pages. Think about removing or hiding parts of your profile that may bias a potential recruiter</span>
+          <label>Edit your post entirely</label>
```
Comment (henryksloan) [inline src/components/Scenario/index.js]: Why the word "entirely"?

**Review by henryksloan — APPROVED** [review]
Approved after changes

**Comment by henryksloan** [comment]
Can you please:
1) Remove the `*_old` files. These will still be in the repo history, so they're not needed. Also, the lines you added to the .gitignore seem to be unnecessary, as no file with that name exists, nor should it.
2) Fix my previous comments.
3) Fix up the formatting of the JSON file (if you don't have a tool to do so, let me know and I can fix it later).

File: src/components/Scenario/index.js (hunk 1 above)
Comment (namzo198) [reply src/components/Scenario/index.js]: agree -> fixed

**Pull request merged**